lib/grape-swagger/doc_methods/move_params.rb

```diff
@@ -8,15 +8,15 @@ def can_be_moved?(params, http_verb)
           move_methods.include?(http_verb) && includes_body_param?(params)
         end
 
-        def to_definition(params, route, definitions)
+        def to_definition(path, params, route, definitions)
```

Comment:
Wouldn't it be better to append the new param `path` instead of prepending it? This is a public method so the change could break other code.